examples/svm/plot_rbf_parameters.py

```diff
@@ -160,7 +160,7 @@ def __call__(self, value, clip=None):
 classifiers = []
 for C in C_2d_range:
     for gamma in gamma_2d_range:
-        clf = SVC(C=C, gamma=gamma)
+        clf = SVC(C=C, gamma=gamma, random_state=42)
```

Comment:
It seems that the random state of an `SVC` is ignored when probability is False, which is the default.